src/devices/tuya.ts

```diff
@@ -15872,6 +15872,7 @@ export const definitions: DefinitionWithExtend[] = [
                 [14, "countdown_2", tuya.valueConverter.raw],
             ],
         },
+        whiteLabel: [tuya.whitelabel("Nova Digital", "ZVL-DUAL", "Valvula Inteligente de Irrigação com saida dupla", ["_TZE284_iilebqoo"])],
```

Comment:
```suggestion
        whiteLabel: [tuya.whitelabel("Nova Digital", "ZVL-DUAL", "Valvula Inteligente de Irrigação com saida dupla", ["_TZE284_iilebqoo"])],
```

Please make the description englisch

Reply:
Done! Thx.